lively.lang/Path.js

```diff
@@ -66,7 +66,7 @@ Object.assign(Path.prototype, {
     // Does the Path resolve to a value when applied to `obj`?
     if (this.isRoot()) return true;
     const parent = this.get(obj, -1);
-    return parent && parent.hasOwnProperty(this._parts[this._parts.length - 1]);
+    return parent && Object.hasOwnProperty.bind(parent)(this._parts[this._parts.length - 1]);
```

Comment:
Why is this necessary? This is hard to read :/

Reply:
If you create a javascript object like `Object.create(null)` and make it encounter this piece of code, it crashes. I have not exactly figured out why these kinds of object float around the system, but making the code more resilient like this, fixed the issues.